include/SW_netCDF.h

```diff
@@ -0,0 +1,62 @@
+#ifndef SWNETCDF_H
+#define SWNETCDF_H
+
+#include "include/SW_datastructs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* =================================================== */
+/*                  Local Definitions                  */
+/* --------------------------------------------------- */
+
+#define DOMAIN_NC 0 // Domain index
+
+// Open netCDF identifer - this should always be 65536 because
+// the program will always have at most one netCDF open
+#define OPEN_NC_ID 65536
+
+// Constants for number of dimensions
+#define ZERODIMS 0
+#define ONEDIM 1
+#define TWODIMS 2
+#define THREEDIMS 3
+
+// Constant sizes
+#define NUMBNDS 2
+
+#define SIZEONE 1
+#define SIZETWO 2
+
+void SW_NC_read_files(PATH_INFO* PathInfo, LOG_INFO* LogInfo);
+int nc_key_to_ID(char* key);
```

Comment:
Why is the module "SW_netCDF" exporting all these functions? Most/all are only called from within the module. I think those should be static, i.e., not exported